sdk/keyvault/azure-keyvault-certificates/tests/test_examples_certificates_async.py

```diff
@@ -4,17 +4,20 @@
 # ------------------------------------
 import asyncio
 import functools
+import pytest
```

Comment:
Grouping pytest with standard library imports contravenes the [PEP 8](https://www.python.org/dev/peps/pep-0008/#imports) style we usually enforce with pylint (but not on test files so I'm nitpicking here)